src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.TranslationAction_Actions.cs

```diff
@@ -257,7 +305,11 @@ public override GeneratorDriver TransformGeneratorDriver(GeneratorDriver generat
                 }
             }
 
-            internal sealed class ReplaceGeneratorDriverAction(GeneratorDriver oldGeneratorDriver, ProjectState newProjectState) : CompilationAndGeneratorDriverTranslationAction
```

Comment:
a few actions were already passing along the newProjectState.  so it's not a stretch to just have all of htem do this.